### ISSUE TITLE
Rename?

### ISSUE THREAD

**Issue by gleckler1** [PCMDI/CIA#4]
@zshaheen @doutriaux1 @chengzhuzhang @durack1 @lee1043 

OK, it looks like we need a rename to keep from getting the wrong type of attention.   How about: 

CISESMs    (Common Input Syntax for ESMs) pronounced  zi-sems ?

**Comment by durack1** [comment]
How about `CHISAM` (Common Hierarchical Input Syntax for the Analysis of Models), I think the ESMs stuff is way too specific - it can also be pronounced "SHAZAM", kinda cool

**Comment by doutriaux1** [comment]
@durack1 I agree with your comment on ESM, not sure about the accronym though, what about Incredible System for Input Syntax? But then again we might get the wrong kind of attention 😄

**Comment by doutriaux1** [comment]
Hierarchical Input Syntax System (HISS)

**Comment by durack1** [comment]
CHISAIM (**C**ommon **H**ierarchical **I**nput **S**yntax for the **A**nalysis and **I**ntercomparison of **M**odels)

Also pronounce-able "SHAZAAIM" - kinda like SHAZAM

If it wasn't obvious, the "..**A**nalysis and **I**ntercomparison.." bit fits nicely with the PCMDI acronym

Or CHISDAIM (**C**ommon **H**ierarchical **I**nput **S**yntax for the **D**iagnosis, **A**nalysis and **I**ntercomparison of **M**odels)

**Comment by doutriaux1** [comment]
Organized Argument Syntax for Input Scripts (OASIS)

**Comment by doutriaux1** [comment]
Syntax Input Nomenclature (SIN)

**Comment by doutriaux1** [comment]
Comprehensive Regulation for Arguments Project (CRAP)

**Comment by lee1043** [comment]
I agree with @durack1 that ESM might be too specific. Like @doutriaux1 first idea as well. haha

**Comment by gleckler1** [comment]
What is "hierarchical" about what we are trying to do?

**Comment by durack1** [comment]
@gleckler1 got a better **H** to replace it with?

And as a counter point, I believe the grand plan is to build out a hierarchical structure which is partitioned by `mip_era`, `realm`, ... along with the PMP version, model versions, obs versions.. If that's not going to be hierarchically organized, I have no idea how we'll do so otherwise

**Comment by lee1043** [comment]
@gleckler1 you may want to consider excluding OASIS from candidates, since it is well-known as a climate model coupler.
